crowdstrikeoauthapi.json

```diff
@@ -14808,10 +14811,9 @@
             ],
             "versions": "EQ(*)",
             "render": {
-                "width": 12,
-                "title": "Device scrolls",
-                "type": "table",
-                "height": 5
+                "title": "Device scroll",
+                "type": "custom",
+                "view": "crowdstrike_view.display_view"
```

Comment:
Seeing that the custom view is using `<table>`, would it be simpler to use the default table?

Reply:
I considered using default table but other actions that also use similar parameters like sort, limit used the custom view as well which show the the parameters as another widget. 
![Screen Shot 2022-05-03 at 1 49 35 PM](https://user-images.githubusercontent.com/32378564/166568605-f46c704b-a841-437e-94f4-168a415cde18.png)

Reply:
Resolved as per our discussion on Slack